travvy/package.json

```diff
@@ -24,7 +24,8 @@
     "babel-eslint": "^10.1.0",
     "eslint": "^6.8.0",
     "eslint-plugin-vue": "^6.2.2",
-    "vue-template-compiler": "^2.6.11"
+    "vue-template-compiler": "^2.6.11",
+    "vue2-google-maps": "^0.10.7"
```

Comment:
Interesting to see that this needs to be added in package.json - nice work!

Reply:
Yeah. I used an npm extension called vue2-google-maps which makes the integration a lot easier